code/online_challenges/src/hackerrank/grid_search/grid_search.cpp

```diff
@@ -0,0 +1,65 @@
+#include <iostream>
+
+int main() 
+{
+    int T, i;
+    std::cin >> T;
+    for (i = 1;i <= T; ++i) 
+    {
+        int R, C, r, c, flag = 0, final = 0;
```

Comment:
Could you please, PLEASE, not use one letter variable names? It is _very_ hard to find out what is being referred to when reading through the code. For example rows and columns.